src/Listener/PodeRequest.cs

```diff
@@ -292,108 +316,23 @@ public async Task<bool> Receive(CancellationToken cancellationToken)
                 PodeHelpers.WriteException(ex, Context.Listener, PodeLoggingLevel.Error);
                 Error = ex;
             }
+            catch (ObjectDisposedException ex)
+            {
+                PodeHelpers.WriteException(ex, Context.Listener, PodeLoggingLevel.Error);
+            }
```

Comment:
I would just let this fall into the `catch (Exception ex)` block below, as that has the same logging but also sets the status code to 500.